src/main.rs

```diff
@@ -675,30 +538,139 @@ async fn main() {
             .unwrap_or(172800),
         beaconchain_api_key: env::var("BEACONCHAIN_API_KEY").unwrap_or("".to_owned()),
     });
+    tracing::info!("Config: {shared_state:?}");
```

Comment:
Should not log credentials to logs!

Reply:
Good catch. I should probably also remove the Beaconcha api key from code, since it won't be needed.

Reply:
(it's also the only secret in this shared state)